lib/libutee/include/tee_bench.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2014, Linaro Limited
+ *
+ * This software is licensed under the terms of the GNU General Public
+ * License version 2, as published by the Free Software Foundation, and
+ * may be copied, distributed, and modified under those terms.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
+#ifndef TEE_BENCH_H
+#define TEE_BENCH_H
+
+#include <inttypes.h>
+
+#define UNUSED(x) (void)(x)
+
+/* max amount of timestamps */
+#define TEE_BENCH_MAX_STAMPS	10
```

Comment:
What does 10 actually mean? That you can add 10 time stamps and that's it and then it wraps around in the ringbuffer. Seems like a low number if that is the case, or?